Eatery/Views/Eateries/PopularTimesView.swift

```diff
@@ -0,0 +1,205 @@
+//
+//  PopularTimesView.swift
+//  Eatery
+//
+//  Created by William Ma on 10/10/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import SnapKit
+import UIKit
+
+/**
+ Why is this layout delegate needed?
+
+ To animate the expansion of the popular times view, we must change the frame
+ of the scroll view. The popular times view is able to manage its own
+ constraints to change its own height. However, without direct access to the
+ outer scroll view (in CampusEateryMenuViewController), it is unable to
+ cleanly animate the change.
+
+ This delegate is the cleanest (albeit still not super clean) way to access the
+ outer scroll view.
```

Comment:
really minor, but isn't the normal convention:

```
/**
 * asdf
 *
 */
```

Reply:
Hmmn, I did a bit of looking and apparently Apple uses `///` on every line to do their documentation. 

e.g. https://github.com/apple/swift/blob/master/stdlib/public/core/String.swift

Xcode parses the documentation cleanly regardless of the style of comment. Any thoughts on standardized comments? I don't think it matters all that much; it's consistency for consistency's sake.